.circleci/config.yml

```diff
@@ -4,8 +4,8 @@ version: 2.1
 stdenv: &stdenv
   environment:
     GOCACHE: &gocache /tmp/go-build
-    IMAGE: &image quay.io/crio/circle-1.12:1.16.0
-    IMAGELEGACY: &imagelegacy quay.io/crio/circle-1.10:1.16.0
+    IMAGE: &image quay.io/crio/circle-1.12:1.16.1
+    IMAGELEGACY: &imagelegacy quay.io/crio/circle-1.10:1.16.1
```

Comment:
Can we use the same test images like for the master branch?

Reply:
fixed

Reply:
though one day I think we should have the images follow the release version (like master is 1.17.x where x is the image version) so we know which crictl is inside and such